api/src/middlewares/projectMiddleware.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from 'express';
+import { ProjectRepository } from '../database/repositories/ProjectRepository';
+import { Project } from '../database/entities/Project';
+import { verifyOrganizationAccess } from './organizationMiddleware';
+
+export const verifyProjectVisibility = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
We don't have explicit access to project yet. So if a project is private, you need to have access to the organization. Otherwise you cannot view the project. Hence named it different.